recipes/fusepy/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "2.0.4" %}
+
+package:
+  name: fusepy
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/f/fusepy/fusepy-{{ version }}.tar.gz
+  sha256: 10f5c7f5414241bffecdc333c4d3a725f1d6605cae6b4eaf86a838ff49cdaf6c
+
+build:
+  number: 0
+  noarch: python
```

Comment:
The package is `noarch` but it it uses `ctypes` to call `libfuse`, not sure if that is present on `Windows` and `OS X` and what is its name there.